src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -72,10 +72,13 @@ export default class ConsentContinuationOut extends Mixins(CountriesProvincesMix
 
   /** Check if Consent is Expired. */
   get isConsentExpired (): boolean {
-    const date = DateUtilities.apiToDate(this.filing.data?.consentContinuationOut?.expiry)
-    const daysToExpire = DateUtilities.daysBetweenTwoDates(new Date(), date)
-    if (isNaN(daysToExpire) || daysToExpire < 0) {
-      return true
```

Comment:
The bug occurred when the consentContinuationOut object did not have an expiry:
- `date` (line 75) was null
- `daysToExpire` (line 76) was NaN
- the method returned True (line 77)